exporting/graphite/graphite.c

```diff
@@ -37,8 +37,10 @@ int init_graphite_instance(struct instance *instance)
         error("EXPORTING: cannot create buffer for graphite exporting connector instance %s", instance->config.name);
         return 1;
     }
-    uv_mutex_init(&instance->mutex);
-    uv_cond_init(&instance->cond_var);
+    if (uv_mutex_init(&instance->mutex))
```

Comment:
There is a buffer created in L35

Reply:
@vlvkobal I understood that we would also take care of these buffers in another issue related to clean the whole exporting variables, am I missing something?

Thank you @stelfrag to report this.

Reply:
Yes, You are right.

Reply:
Thank you @vlvkobal !
@stelfrag , please, ignore the current error, because we will deal with it in another PR.